source/server/spongineer/logs.rst

```diff
@@ -9,6 +9,128 @@ logfiles from SpongeForge and SpongeVanilla servers including short descriptions
    :depth: 2
    :local:
 
+Configure Logging
+=================
+
+Sometimes plugins log too much information and thus spam the logs with useless messages and thus hide more important
+information. It is also possible to configure the logging in a way to split the logs in two or more separate files. One
+could be optimized for the moderators that pay attention to their users' activity/behavior and other logs could be used
+to monitor important or new plugins that are important to the admins.
+
+The simplest way to configure the logging is modifying the ``log4j2.xml`` configuration that will be used by
```

Comment:
never do this, ever.. you can pass a custom log4j `-Dlog4j.configurationFile=log4j2.xml` the last path being the path to your custom file